From 94efd93e2bdd2a1d739f4f86bf6a013b081a481a Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Wed, 9 Jun 2010 20:32:26 -0400
Subject: [PATCH] tc: revert "echo" in install target

The recent commit "iproute2: add option to build m_xt as a tc module"
(ab814d635529787) looks like it wrongly included debug changes in the
install target.  So drop the `echo` so the tc binary actually gets
installed again.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 tc/Makefile |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/tc/Makefile b/tc/Makefile
index 01a16fc..3aa9f26 100644
--- a/tc/Makefile
+++ b/tc/Makefile
@@ -99,10 +99,10 @@ libtc.a: $(TCLIB)
 	$(AR) rcs $@ $(TCLIB)
 
 install: all
-	echo mkdir -p $(MODDESTDIR)
-	echo install -m 0755 tc $(DESTDIR)$(SBINDIR)
+	mkdir -p $(MODDESTDIR)
+	install -m 0755 tc $(DESTDIR)$(SBINDIR)
 	for i in $(TCSO); \
-	do echo install -m 755 $$i $(MODDESTDIR); \
+	do install -m 755 $$i $(MODDESTDIR); \
 	done
 	if [ ! -f $(MODDESTDIR)/m_ipt.so ]; then \
 	if [ -f $(MODDESTDIR)/m_xt.so ]; \
-- 
1.7.1

